schema.graphql

```diff
@@ -493,15 +495,23 @@ type Indexer @entity {
   delegators: [DelegatedStake!]! @derivedFrom(field: "indexer")
   "Total tokens delegated to the indexer"
   delegatedTokens: BigInt!
+  "Ratio between the amount of the indexers own stake over the total usable stake."
+  ownStakeRatio: BigDecimal!
+  "Ratio between the amount of delegated stake over the total usable stake."
+  delegatedStakeRatio: BigDecimal!
   "Total shares of the delegator pool"
   delegatorShares: BigInt!
   "Exchange rate of of tokens received for each share"
   delegationExchangeRate: BigDecimal
-  "The percent the Indexer agrees to share with delegators. In parts per million"
+  "The percent of indexing rewards generated by the total stake that the Indexer keeps for itself. In parts per million"
   indexingRewardCut: Int!
+  "The percent of indexing rewards generated by the delegated stake that the Indexer keeps for itself"
+  indexingRewardEffectiveCut: BigDecimal!
+  "The percent of reward dilution delegators experience because of overdelegation. Overdelegated stake can't be used to generate rewards but still gets accounted while distributing the generated rewards. This causes dilution of the rewards for the rest of the pool."
```

Comment:
description seems long. maybe `The percent of rewards dilution that delegators experience from overdelegation to the indexer.` 

also i think we might be able to just call it `overDelegationDilution` wdyt?

Reply:
Yeah, could be simplified in both naming and description. I wanted to add as much info as possible since most delegators think these metrics are too complex

Reply:
How about we keep the long comment, but shorten the name?